package.json

```diff
@@ -22,14 +22,20 @@
         "concurrently": "3.5.1",
         "husky": "0.14.3",
         "protractor": "5.2.2",
-        "typescript": "2.6.2"
+        "typescript": "2.6.2",
+        "rollup": "0.52.3",
```

Comment:
when this shapes up more, the user shouldn't be exposed to the rollup npm dependencies